swarm/network/README.md

```diff
@@ -57,7 +57,7 @@ receipts for a deleted chunk easily to refute their challenge.
 - syncing should be resilient to cut connections, metadata should be persisted that
 keep track of syncing state across sessions, historical syncing state should survive restart
 - extra data structures to support syncing should be kept at minimum
-- syncing is organized separately for chunk types (resource update v content chunk)
+- syncing is organized separately for chunk types (Swarm Feed Updates v regular content chunk)
```

Comment:
this is not true. it is handled not separately

Reply:
changed to: *syncing is **not** organized separately for chunk types (Swarm feed updates v regular content chunk)*

However, take into account I was just changing terms here, not really changing the meaning. `swarm/network` owner should be aware of this.